src/richie/apps/core/templates/richie/base.html

```diff
@@ -66,48 +66,60 @@
 
         {# Wrap all body content in an ID'd element to easily remove it from view (using aria-hidden) #}
         {# when a modal is open. This is important for accessibility. #}
+
+        {# First link on a website should always be a skip to content #}
+        {# Access key directive allows users to use keyboard shortcuts #}
+        <a href="#site-content" class="sr-only sr-only-focus" accesskey="c">{% trans "Skip to main content" %}</a>
```

Comment:
I think this should be inside `#modal-exclude`? `#modal-exclude` is used to remove content while showing a modal, so I'm not sure the link would even do anything while a modal is open

Reply:
I'll review that.